sklearn/decomposition/kernel_pca.py

```diff
@@ -107,10 +107,10 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         `remove_zero_eig` are not set, then all components are stored.
 
     dual_coef_ : array, (n_samples, n_features)
-        Inverse transform matrix. Set if `fit_inverse_transform` is True.
+        Inverse transform matrix. Only available when `fit_inverse_transform` is True.
 
     X_transformed_fit_ : array, (n_samples, n_components)
-        Projection of the fitted data on the kernel principal components.
+        Projection of the fitted data on the kernel principal components. Only available when `fit_inverse_transform` is True.
```

Comment:
please break the line and make sure it's not longer than 79chars. Also, single backticks don't do anything and you should use double backticks. Finally, if you say "closes #11126" or "fixes #11126" in the issue description, the issue will automatically be closed on merge.